src/main/java/com/dev/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,30 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.exceptions.AuthenticationException;
+import com.dev.cinema.exceptions.RegistrationException;
+import com.dev.cinema.model.dto.UserDto;
+import com.dev.cinema.security.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/registration")
+    public void register(@RequestBody UserDto userDto) throws AuthenticationException {
+        if (!userDto.getPassword().equals(userDto.getRepeatedPassword())) {
+            throw new RegistrationException("The passwords should be equal");
```

Comment:
not sure we need to throw an exception here. We should pass a message to the client. Anyway, we will use annotation for this purpose in the future homework